docs/projects/ada4355_fmc/index.rst

```diff
@@ -66,6 +66,32 @@ The data path and clock domains are depicted in the below diagram:
    :align: center
    :alt: ADA4355_FMC/ZedBoard block diagram
 
+Configuration modes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The BUFMRCE_EN configuration parameter defines the type of evaluation board
+used, which are differentiated by how the frame clock signals are distributed.
+For the older evaluation board, the XDC constraints are not optimized for
+ISERDES, as the frame clock signals are located in a different I/O bank from
+the other related signals. To address this, a BUFMRCE buffer is used to
+distribute the frame clock to all ISERDES instances.
+
+By default it is set to 0. Depending on the type of evaluation board, some
+hardware modifications need to be done on the board and/or ``make`` command:
+
+In case of the Eval-Board with optimized xdc:
+
+.. shell:: bash
+
+   $make BUFMRCE_EN=0
+
+In case of the Eval-Board with non optimized xdc:
+
+.. shell:: bash
+
+   $make BUFMRCE_EN=1
+
+
```

Comment:
Could you give some details on how to differentiate these boards? Are they named with rev. A/B? If not, are there any distinctive marks on the evaluation board?

```suggestion
By default, BUFMRCE_EN is set to 0. Depending on the type of evaluation board, some
hardware changes need to be done on it and/or for the `make` command:

In case of the newer version of the board (with optimized constraints):

.. shell:: bash

   ~/hdl/projects/ada3455_fmc/zed
   $make BUFMRCE_EN=0

In case of the older version of the board (with non-optimized constraints):

.. shell:: bash

   ~/hdl/projects/ada3455_fmc/zed
   $make BUFMRCE_EN=1

```